clarity-ext-scripts/clarity_ext_scripts/covid/fetch_biobank_barcodes.py

```diff
@@ -128,3 +136,20 @@ def _plate_barcode_from(self, biobank_matrix):
 
     def _biobank_key(self, well, plate_barcode):
         return '{}_{}'.format(well, plate_barcode)
+
+    # TODO this has been duplicated from `validate_sample_creation_list`
+    #      fix this later.
+    def get_raw_sample_list(context):
```

Comment:
there is a missing "self" here

Reply:
This is just copied from the other script, so I also thought about that, but I don't know how these things should be setup.